steps/browserstack-bitrise-espresso-step/1.0.0/step.yml

```diff
@@ -0,0 +1,256 @@
+title: BrowserStack App Automate - Espresso
+summary: |
+  Run your Espresso tests on BrowserStack App Automate
+description: |
+  Run your Espresso tests on BrowserStack App Automate. This step collects the built APK from $BITRISE_APK_PATH  and test apk from `$BITRISE_TEST_APK_PATH` Environment Variables
+website: https://www.browserstack.com/app-automate
+source_code_url: https://github.com/browserstack/browserstack-bitrise-espresso-step
+support_url: https://github.com/browserstack/browserstack-bitrise-espresso-step/issues
+published_at: 2022-07-28T14:39:20.868019+05:30
+source:
+  git: https://github.com/browserstack/browserstack-bitrise-espresso-step.git
+  commit: 22741ea5a75c0f5b2b731a0e35d95629aa326a2c
+host_os_tags:
+- osx-10.10
+- ubuntu-16.04
+project_type_tags:
+- android
+type_tags:
+- test
+toolkit:
+  go:
+    package_name: github.com/browserstack/browserstack-bitrise-espresso-step
+deps:
+  brew:
+  - name: git
+  - name: wget
+  apt_get:
+  - name: git
+  - name: wget
+is_requires_admin_user: true
+is_always_run: false
+is_skippable: false
+run_if: ""
+inputs:
+- browserstack_username: null
+  opts:
+    category: Authentication
+    description: Username of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Username for the BrowserStack account
+    title: BrowserStack username
+- browserstack_accesskey: null
+  opts:
+    category: Authentication
+    description: Access Key of the BrowserStack account
+    is_required: true
+    is_sensitive: true
+    summary: Accesskey for the BrowserStack account
+    title: BrowserStack accesskey
+- android_app_under_test: null
+  opts:
+    category: App & Test Suite
+    description: Path of the app (.apk) file
+    is_expand: true
+    is_required: true
+    summary: Path to the app .apk file
+    title: Android app under test
+- espresso_test_suite: null
+  opts:
+    category: App & Test Suite
+    description: Path of the test suite (.apk) file
+    is_expand: true
+    is_required: true
+    summary: Path to the testsuite .apk file
+    title: Espresso test suite
+- devices_list: null
+  opts:
+    category: Devices
```

Comment:
If a category has only a single input, we suggest not using a category at all, it looks cleaner on the UI (unless you have plans for adding more device-related inputs)